File: DecisionLogica/Universidad.cpp
#include <iostream>
using namespace std;

string nombre;
int nota1;
int nota2;
int nota3;
string estudio;
float promedio;
int Vprecreditos = 50000;
int Vposcreditos = 300000;
int creditos;
float total;
int main() {
	cout<< "Nombre del alumno: "<<endl;
	cin >> nombre;
	cout<< "notas del alumno: "<< endl;
	cin >> nota1;
	cin >> nota2;
	cin >> nota3;
	cout << "estudio superior del estudiante: " << endl;
	cin >> estudio;
	promedio = (nota1 + nota2 + nota3)/3;
	cout << promedio << endl;
	if (promedio >= 4.5) (estudio="pregrado");
  {
    creditos= 28;
		total = Vprecreditos*creditos*0.25;
		cout << total << endl;
  } 
	if (((promedio >= 4.0) && (promedio < 4.5 && estudio="pregrado"))
  {
    creditos= 25;
		total = Vprecreditos*creditos*0.10;
		cout << total << endl;
  }
	if (promedio>3.5) (promedio<4.0)  (estudio="pregrado");
  {
		creditos=20;
		total=Vprecreditos*creditos,
		cout << total << endl;
  }

	if (promedio>=2.5) (promedio<3.5) (estudio="pregrado");
  {
    creditos=15;
		total=Vprecreditos*creditos,
		cout << total << endl;
  }
	if (promedio<2.5) (estudio="pregrado");
  {
    cout << "no podra graduarse" << endl;
  }
	if (promedio >= 4.5) (estudio="posgrado");
  {
    creditos=20;
    total=Vposcreditos*creditos*0.2;
    cout<< total<< endl;
  }
	if (promedio < 4.5) (estudio="posgrado");
  {
    creditos=10;
    total=Vposcreditos*creditos;
    cout << total << endl;
  }

}

File: DecisionLogica/Universidad.py
promedio1 = float
promedio = float
Vprecreditos = 50000
Vposcreditos = 300000
creditos = int
total = float
3
print ("notas del alumno (solo 3): ")
nota1 = float(input("ingrese su primera nota: "))
nota2 = float(input("ingrese su segunda nota: "))
nota3 = float(input("ingrese su tercera nota: "))
promedio1= nota1 + nota2 + nota3
promedio = promedio1 / 3
print ("estudio superior del estudiante: ")
desicion = str(input())
print (promedio)
if (desicion == "pregrado"):
    if (promedio >= 4.5) :
        creditos= 28
        total = Vprecreditos*creditos*0.25
        print (total)
    else:
        if (promedio >= 4.0) and (promedio < 4.5):
            creditos= 25
            total = Vprecreditos*creditos*0.10
            print (total) 
        else:
            if (promedio>3.5) and (promedio<4.0):
                creditos=20
                total=Vprecreditos*creditos
                print (total)
            else:
                if (promedio>=2.5) and (promedio<3.5):
                    creditos=15
                    total=Vprecreditos*creditos
                    print (total)
                else:
                    if (promedio<2.5):
                        print ("no podra graduarse")
else:
    if (desicion == "posgrado"):
        if (promedio >= 4.5):
            creditos=20
            total=Vposcreditos*creditos*0.2
            print (total)
        else:
            if (promedio < 4.5):
                creditos=10
                total=Vposcreditos*creditos
                print (total) 
